test/role.test.js

```diff
@@ -287,6 +298,7 @@ describe('role model', function() {
     it('should resolve user by username', function(done) {
       ACL.resolvePrincipal(ACL.USER, user.username, function(err, u) {
         if (err) return done(err);
+
         expect(u.username).to.eql(user.username);
         done();
```

Comment:
can you put a space btwn the dones too? completing the test has nothing to do with the assertion logic.